src/optimize/base_optimizer.js

```diff
@@ -172,26 +192,30 @@ export default class BaseOptimizer {
         devtoolModuleFilenameTemplate: '[absolute-resource-path]'
       },
 
-      plugins: [
-        new ExtractTextPlugin('[name].style.css', {
-          allChunks: true
-        }),
+      optimization: {
+        splitChunks: {
+          cacheGroups: {
+            commons: {
+              name: 'commons',
+              chunks: 'initial',
+              minChunks: 2,
+              reuseExistingChunk: true, // check this again
+            }
+          }
+        },
+        noEmitOnErrors: true
```

Comment:
Is this what we want? Seems like some error feedback would be helpful

Reply:
Yeah it is. Actually is our current behaviour for the webpack 3 https://github.com/elastic/kibana/blob/master/src/optimize/base_optimizer.js#L193